### ISSUE TITLE
`stroke`and `stroke-width` properties classified as selectors

### ISSUE THREAD

**Issue by maranomynet** [rossPatton/stylint#254]
This Stylus code

```css
svg line {
   stroke: #555;
   stroke-width: 1;
}
```

triggers this warning

```
line 2: always use brackets when defining selectors   stroke: #555;
line 3: always use brackets when defining selectors   stroke-width: 1;
```
